components/event-block/EventBlockChapters.vue

```diff
@@ -0,0 +1,235 @@
+<template>
+  <div class="event-block-chapters">
+    <p class="event-block-chapters__title">
+      {{ title }}
+    </p>
+
+    <swiper :options="swiperOptions">
+      <swiper-slide
+        v-for="item in items"
+        :key="item.slug"
+      >
+        <nuxt-link
+          :to="`${currentRoute}/${item.slug}`"
+          class="event-block-chapters__link"
+        >
+          <span class="event-block-chapters__item-title">{{ item.title }}</span>
+          <img
+            v-if="item.cover"
+            :data-src="`${item.cover.url}?auto=compress&w=470`"
+            alt=""
+            class="event-block-chapters__item-image swiper-lazy"
+          >
+          <div
+            v-else
+            class="event-block-chapters__item-placeholder"
+          />
+          <div class="swiper-lazy-preloader swiper-lazy-preloader-white" />
+        </nuxt-link>
+      </swiper-slide>
+
+      <div
+        slot="button-prev"
+        class="swiper-button-prev"
+      />
+
+      <div
+        slot="button-next"
+        class="swiper-button-next"
+      />
+
+      <div
+        slot="scrollbar"
+        class="swiper-scrollbar"
+      />
+    </swiper>
+  </div>
+</template>
+
+<script>
```

Comment:
Could you indent the whole `<script>` content? Makes it consistent with the `<template>` and foldable/collapsible for more code editors.